tests/cases/standalone/common/alter/alter_table_alter_column_set_default.result

```diff
@@ -0,0 +1,128 @@
+--- alter table to add new column with default timestamp values aware of session timezone test ---
+CREATE TABLE test1 (i INTEGER, j TIMESTAMP time index, PRIMARY KEY(i));
+
+Affected Rows: 0
+
+ALTER TABLE test1 ADD COLUMN k INTEGER DEFAULT 100;
+
+Affected Rows: 0
+
+ALTER TABLE test1 ADD COLUMN l INTEGER DEFAULT 200;
+
+Affected Rows: 0
+
+ALTER TABLE test1 ADD COLUMN m TIMESTAMP DEFAULT '2024-01-30 00:00:00';
+
+Affected Rows: 0
+
+ALTER TABLE test1 ADD COLUMN n INTEGER DEFAULT 300;
+
+Affected Rows: 0
+
+--- set default value for k, l, n, n ---
+ALTER TABLE test1 ALTER k SET DEFAULT 101;
+
+Affected Rows: 0
+
+ALTER TABLE test1 ALTER l SET DEFAULT 201, ALTER m SET DEFAULT '2024-01-30 00:00:00', ALTER n SET DEFAULT 300;
```

Comment:
Let's use `MODIFY COLUMN [column name]` instead of `alter [column name]` for API consistency, as shown in https://docs.greptime.com/reference/sql/alter/#create-an-index-for-a-column.

Reply:
Vote for MODIFY COLUMN.